source/server/options_impl.cc

```diff
@@ -13,11 +13,6 @@
 #include "spdlog/spdlog.h"
 #include "tclap/CmdLine.h"
 
-// Can be overridden at compile time
-#ifndef ENVOY_DEFAULT_MAX_STATS
```

Comment:
Moved to `options_impl.h` for visibility.

Reply:
Makes sense.